packages/flutter_localizations/lib/src/l10n/material_ar.arb

```diff
@@ -20,6 +20,8 @@
   "previousMonthTooltip": "الشهر السابق",
   "nextPageTooltip": "الصفحة التالية",
   "previousPageTooltip": "الصفحة السابقة",
+  "firstPageTooltip": "الأولى",
+  "lastPageTooltip": "الاخيرة",
```

Comment:
Shouldn't these be the English placeholders?

Reply:
Based on your other comment that you didn't take community translations, yes! I actually took some of the translations from another open source project as well, but I'll turn them all back to English then

Reply:
Done